.gitignore

```diff
@@ -8,4 +8,5 @@ demo/tmp/
 .rbenv-gemsets
 *.swp
 Gemfile.lock
+test/gemfiles/*.lock
 .ruby-version
```

Comment:
There's a `test/dummy` directory on my machine right now that seems to contain only databases and log files. Could we add this to `.gitignore`. Also, for those of us who use Vagrant, would you mind adding `.vagrant` and `Vagrantfile` to `.gitignore`?

Reply:
Sure, I'll add the Vagrant stuff. As far as `test/dummy` goes, that directory was a byproduct of the old location of the test Rails app, which as since moved to `demo/`. You can safely remove the `test/dummy` directory and those files will not come back.